.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Bug report'
+labels: bug
+---
+
+## Prerequisites
+
+Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**
+
+- [ ] I am running the latest version
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+
+## Expected Behavior
+
+Please describe the behavior you are expecting
+
+## Current Behavior
+
+What is the current behavior?
+
+### Steps to Reproduce
+
+Please provide detailed steps for reproducing the issue.
+
+1. step 1
+2. step 2
+3. ...
+
+### Context
+
+Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.
+
+* Library version: X.Y.Z
+* Platform: Android/iOS.
+* Platform logs (logcat/XCode):
```

Comment:
Perhaps inform the user how to set logs correctly?

Reply:
I guess you mean Android logcat's tutorial and the same for iOS?

Reply:
`RxBleClient.setLogLevel()` — I am not sure how it is managed in the RN part

Reply:
Added link to the docs and comment at the paragraph above the list.